docs/ark_integration.md

```diff
@@ -0,0 +1,50 @@
+# Kubicorn and ARK integration
+
+## Introduction
+This document proposes integration between Kubicorn and ARK, Kubicorn provides infrastructure management and ARK provides Kubernetes cluster resources and applications backup and recovery mechanism. Both the tools are recent addition to the Kubernetes ecosystem and are still evolving. Users of either tools are looking for better integrations, which can provide a seamless solution for backup / restore of the complete infra + apps.
+
+
+## Backup Scenarios
+
+Two Scenarios for doing backups, Kubernetes cluster resources, applications (ARK) and infrastructure (Kubicorn)
+
+**Scenario 1** - Backup for any existing Kubernetes cluster existing applications and infrastructure:
+ARK provides you with a mechanism to snapshot with highly customizable settings. We can add this functionality to Kubicorn, given any running Kubernetes cluster. User should be able to run something like -
+
+                `kubicorn backup -f <kube.cofig>`
```

Comment:
What do we pass in for `-f` here? The kube config?

Reply:
I am wondering if we want to stick with the parlance of using `name` to identify/look up cluster information. It's what the other commands use as an identifier.

Maybe support both?